tests/unit/test_deprecations.py

```diff
@@ -40,10 +37,13 @@ def test_mx8_not_maintained(self):
         assert not runtime.is_version_maintained(MXVersion("8.17"))
 
     def test_mx9_maintained(self):
-        assert runtime.is_version_maintained(MXVersion("9.6.9"))
-        assert runtime.is_version_maintained(MXVersion("9.12.1"))
```

Comment:
Could you move these to `test_mx9_not_maintained`?

Reply:
done